code/src/Components/ThirdQuestion.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Button from "./Button";
+
+const ThirdQuestion = ({
```

Comment:
These checkboxes are acting as radio buttons right now and the summary is not showing any of this input. Take another look at Maks' lecture/code about it -> https://technigo.slack.com/archives/C017W9LGX3R/p1603279217488000

Reply:
This was a tough one! Had a 1:1 with Max with no solution in the 20 minutes we had. Discovered after that it was a line in the button that had messed it up. Now it works as intended 😅

Reply:
Great!